@ecomplus/storefront-template/template/scss/components/_banner-slider.scss

```
@@ -11,6 +11,11 @@
     max-height: calc(80rem * var(--banner-aspect-ratio-mobile, --banner-aspect-ratio));
   }
 
+  &--full-width {
+    max-width: 100vw;
+    max-height: none;
```

```suggestion
    max-width: 160rem;
    max-height: calc(160rem * var(--banner-aspect-ratio));
```

na verdade deixar sempre em 100vm também não me parece muito bom, em telas grandes demais ficar ruim, limitando para 160rem (= 2560px = QHD)